backend/clap/src/main/java/zpi/ppea/clap/service/GrpcServerMock.java

```diff
@@ -1,23 +1,186 @@
 package zpi.ppea.clap.service;
 
-import backend.clap.HelloReply;
-import backend.clap.HelloRequest;
-import backend.clap.MyServiceGrpc;
+import data_store.*;
 import io.grpc.stub.StreamObserver;
 import lombok.extern.slf4j.Slf4j;
 import org.lognet.springboot.grpc.GRpcService;
 
+import java.util.Arrays;
+
 @Slf4j
 @GRpcService
-public class GrpcServerMock extends MyServiceGrpc.MyServiceImplBase {
+public class GrpcServerMock extends DataStoreGrpc.DataStoreImplBase {
```

Comment:
The Mock server is not needed as this is supposed to be integrated from now on

Reply:
It will be easier and faster for local testing, I will put it to gitignore.